components/03-sections/01-accordion/accordion.hbs

```diff
@@ -1,15 +1,22 @@
 <section class="select-publication-sec scholarship-type-designs">
     <div class="container">
-        <div id="accordion">
-            <div class="card">
-                <div class="card-header"> <a class="card-link" data-bs-toggle="collapse"
-                        href="#collapse-01">{{ heading }}</a>
-                </div>
-                <div id="collapse-01" class="collapse" data-bs-parent="#accordion">
+        <div class="accordion" id="accordion">
+            <div class="accordion-item">
+                <h5 class="accordion-header" id="heading-01">
+                    <button type="button" class="accordion-button collapsed" data-bs-toggle="collapse"
+                        data-bs-target="#collapse-01">
+                        {{heading}}
+                    </button>
+                </h5>
+                <div id="collapse-01" class="accordion-collapse collapse" data-bs-parent="#accordion">
                     <div class="card-body">
                         <div class="accordion-body-content">
```

Comment:
The content inside the body looks a little tight and might need a little padding. Maybe add a `p-3` class or consult with @Uri-E or @garza-utsa on their thoughts.

Reply:
I added these styles from orginal scss for **.accordion-body-content** in last commit.  It was originally named **.accordion-body-cntnt** and I missed it when I change cntnt to content for all other accordion selectors. So it wasn't picking it up first time around.  

Long story short.

this was fixed in the last commit I made.